app/app/route.js

```diff
@@ -5,11 +5,16 @@ export default Ember.Route.extend({
     if (tokens.length === 0) {
       tokens.push(this.currentModel.get('handle'));
     }
-    var stack = this.currentModel.get('stack');
+
+    let stack = this.get('stack');
+    Ember.assert('App Route must have a stack', stack);
```

Comment:
`Ember.assert('App Route must have a stack', !!stack);` is preferred. It ensures only a boolean is passed, which lets the VM optimize.